install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
I think `nvm_detect_profile` really seems to **guess** which shell be using, the order determinate the only profile when multiple profile existing, maybe there should be a option for shell will be easier for this issue?

```bash
nvm_detect_profile() {
  if [ -f "$PROFILE" ]; then
    echo "$PROFILE"
  elif [ -f "$HOME/.bashrc" ]; then
    echo "$HOME/.bashrc"
  elif [ -f "$HOME/.bash_profile" ]; then
    echo "$HOME/.bash_profile"
  elif [ -f "$HOME/.zshrc" ]; then
    echo "$HOME/.zshrc"
  elif [ -f "$HOME/.profile" ]; then
    echo "$HOME/.profile"
  fi
}
```